packages/angular-cli/blueprints/ng2/files/__path__/test.ts

```diff
@@ -14,21 +19,15 @@ declare var require: any;
 // Prevent Karma from running prematurely.
 __karma__.loaded = function () {};
 
+// First, initialize the Angular testing environment.
```

Comment:
Please surround this by `try { ... } catch(err) { karma.error(err); }` to catch potential framework errors and break the test.